manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -432,3 +433,57 @@ LimitRequestFieldSize 524288
 `
 	return fmt.Sprintf(s, protocol)
 }
+
+func apiHttpdConfig(protocol string) string {
+	s := `
+## ManageIQ HTTP Virtual Host Context
+
+Listen 3000
+
+# Timeout: The number of seconds before receives and sends time out.
+Timeout 120
+ServerSignature Off
+ServerTokens Prod
+
+RewriteEngine On
+Options SymLinksIfOwnerMatch
+
+# LimitRequestFieldSize: Expand this to a large number to allow pass-through.
+#   This does not introduce a potential DoS, because the value is validated by
+#   the httpd container first.  However, if a user changes this value in the
+#   httpd container, we need to be able to accomodate that value here also.
+LimitRequestFieldSize 524288
+
+<VirtualHost *:3000>
+  IncludeOptional conf.d/*_config
+
+  ServerName %s://web-service
+  DocumentRoot /var/www/miq/vmdb/public
+
+  RewriteCond %{REQUEST_URI}     ^/ws/notifications [NC]
+  RewriteCond %{HTTP:UPGRADE}    ^websocket$ [NC]
+  RewriteCond %{HTTP:CONNECTION} ^Upgrade$   [NC]
+  RewriteRule .* ws://localhost:3001%{REQUEST_URI}  [P,QSA,L]
+  ProxyPassReverse /ws/notifications ws://localhost:3001/ws/notifications
+
+  RewriteRule ^/web-service/service(?!/(assets|images|img|styles|js|fonts|vendor|gettext)) /web-service/service/index.html [L]
```

Comment:
I think this is the only line that's different?  But this isn't valid because /web-service/service isn't a path.  The other one is for the SUI.